.readthedocs.yaml

```
@@ -0,0 +1,21 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# RTD API version
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: >-
+      3.11
+  # Build with make coredocs
+  commands:
+    - python -m venv "${READTHEDOCS_VIRTUALENV_PATH}"
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -m pip install --exists-action=w -r tests/requirements.in -c tests/requirements.txt
+    - >-
+      "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python docs/bin/clone-core.py
+    - make coredocs -C docs/docsite
+    - mkdir -pv "${READTHEDOCS_OUTPUT}"/html
+    - cp -rv docs/docsite/_build/html/** "${READTHEDOCS_OUTPUT}"/html
```

`cp` doesn't require `**` as it's an fnmatch construct. You're just copying the contents over. Though I'd do it differently:
```suggestion
    - cp -rv docs/docsite/_build/html "${READTHEDOCS_OUTPUT}"/
```

---

OTOH, I don't understand why you wouldn't just move it as RTD is a throwaway build env that doesn't persist anything temporary.
```suggestion
    - mv -v docs/docsite/_build/html "${READTHEDOCS_OUTPUT}"/
```